reference/solver/batch_bicgstab_kernels.hpp.inc

```diff
@@ -345,28 +283,27 @@ inline void batch_entry_bicgstab_impl(
                rhs_norms_entry, res_norms_entry);
 
     // stopping criterion object
-    StopType stop(nrhs, opts.max_its, opts.residual_tol, rhs_norms_entry.values,
-                  converged);
+    StopType stop(opts.residual_tol, rhs_norms_entry.values);
 
     int iter = -1;
 
     while (1) {
         ++iter;
 
+        // bool all_converged = stop.check_converged(iter,
```

Comment:
commented code, here and in a few other places

Reply:
Ah yes, I've removed them in the next PR which is coming up shortly.